src/core/js/models/configModel.js

```diff
@@ -26,6 +26,9 @@ define([
             // 'configModel:loadCourseData' event starts the core content collections and models being fetched
             this.fetch({
                 success: _.bind(function() {
+                  Adapt.trigger('offlineStorage:prepare');
+                  Adapt.wait.queue(_.bind(function() {
```

Comment:
it's a small point but now we don't need to support stupid old IE browsers we can safely do `function.bind` instead of `_.bind`

I think this looks a lot cleaner:
```
 this.fetch({
	success: function() {
		Adapt.trigger('offlineStorage:prepare');
		Adapt.wait.queue(function() {
			Adapt.trigger('configModel:dataLoaded');
			if (this.get('_canLoadData')) {
				Adapt.trigger('configModel:loadCourseData');
			}
			
		}.bind(this));
	}.bind(this),
	error: function() {
		console.log('Unable to load course/config.json');
	}
});
```